configapi/api_slice_mgmt.go

```diff
@@ -198,15 +200,28 @@ func NetworkSlicePostHandler(c *gin.Context, msgOp int) bool {
 	logger.ConfigLog.Infof("url: %v ", req.URL)
 	procReq := req.Body.(configmodels.Slice)
 
+	for _, gnb := range procReq.SiteInfo.GNodeBs {
+		if !isValidName(gnb.Name) {
+			err := fmt.Errorf("invalid gNB name `%s` in Network Slice %s. Name needs to match the following regular expression: %s", gnb.Name, sliceName, NAME_PATTERN)
+			logger.ConfigLog.Error(err.Error())
+			return err
+		}
+		castedGnbTac := fmt.Sprint(gnb.Tac)
```

Comment:
```suggestion
		castedGnbTac := strconv.Itoa(gnb.Tac)
```

This would be a bit more efficient, but this is very minor.

Reply:
Would not it be better to properly update `configmodels/model_inventory.go` to have `Tac` as a number as follows:
```diff
type PutGnbRequest struct {
-	Tac string `json:"tac"`
+	Tac int32 `json:"tac"`
}
```
Or, alternatively, to update `configmodels/model_slice_site_info_g_node_bs.go` to have `Tac` as string as follows:
```diff
type SliceSiteInfoGNodeBs struct {
	Name string `json:"name,omitempty"`

	// unique tac per gNB. This should match gNB configuration.
-	Tac int32 `json:"tac,omitempty"`
+	Tac string `json:"tac,omitempty"`
}
```
It looks pretty strange to me that we have `Tac` as string in one place while we have `Tac` as int32 in another place